book/skipped.md

```diff
@@ -194,7 +194,7 @@ they have batteries of hundreds of thousands of [unit] and [integration] tests.
 Recently, a lot of focus has been put on robust
 [cross-browser tests](https://wpt.fyi) that allow a single automated test
 to run on all browsers to verify that they all behave the same on the same
-input. And there are now yearly [Interoperability]^["Interop", for short.]
+input. And there are now yearly [interoperability]^["Interop", for short.]
```

Comment:
Interop is a proper noun because it's the specific benchmark; interoperability is a concept.